usr/src/cmd/zfs/zfs_main.c

```diff
@@ -2330,6 +2331,9 @@ us_compare(const void *larg, const void *rarg, void *unused)
 			if (rv64 != lv64)
 				rc = (rv64 < lv64) ? 1 : -1;
 			break;
+
+		default:
+			break;
```

Comment:
If it's some other prop, we ignore the request to sort by it?  Or is it impossible to request other properties, in which case we should assert?